lib/walex/replication/server.ex

```diff
@@ -152,8 +159,18 @@ defmodule WalEx.Replication.Server do
   def handle_data(<<?k, wal_end::64, _clock::64, reply>>, state) do
     messages =
       case reply do
-        1 -> [<<?r, wal_end + 1::64, wal_end + 1::64, wal_end + 1::64, current_time()::64, 0>>]
-        0 -> []
+        1 ->
+          Logger.debug(
+            "standby status update, remote wal: #{wal_end} current wal: #{state.wal_position}"
+          )
+
+          [
+            <<?r, state.wal_position::64, state.wal_position::64, state.wal_position::64,
```

Comment:
this needed to be the `wal_position+1` I think
<img width="1234" alt="Screenshot 2024-12-30 at 16 08 25" src="https://github.com/user-attachments/assets/708aceb5-dc14-488a-bb12-bc86b2202837" />